sdks/java/extensions/sql/shell/build.gradle

```diff
@@ -37,3 +37,7 @@ processResources {
 }
 
 mainClassName = "org.apache.beam.sdk.extensions.sql.jdbc.BeamSqlLine"
+
```

Comment:
Can we add comment `./gradlew :beam-sdks-java-extensions-sql-shell:run --console plain -q --no-daemon` as an example of how to run BeamSqlLine from gradle?

Reply:
Great suggestion. Updated.